### PR TITLE
chore: replace ROCKs with rocks

### PR DESCRIPTION
**Pull request by DnPlas**
This commit replaces ROCKs with rocks to be in sync with Canonical's standard terminology.

Part of canonical/bundle-kubeflow#916

### CODE REVIEW

File: mlserver-huggingface/rockcraft.yaml [hunk 1]
```diff
@@ -43,10 +43,10 @@ parts:
       ./hack/build-wheels.sh ${CRAFT_PART_INSTALL}/opt/mlserver/dist
 
     override-stage: |
-      export ROCK_RUNTIME="huggingface"
+      export rock_RUNTIME="huggingface"
```
Comment (misohu) [inline mlserver-huggingface/rockcraft.yaml:46]: I dont think we should change the env variables. same comment for changes bellow :)

Comment (DnPlas) [reply mlserver-huggingface/rockcraft.yaml:46]: Whoops, the script replaced this :P

**Review by misohu — COMMENTED** [review]
LGTM :) CI is failing because of this https://github.com/canonical/charmed-kubeflow-workflows/issues/53

**Review by misohu — DISMISSED** [review]

**Comment by DnPlas** [comment]
@misohu now the CI will fail because the rocks are written in an old format, (they still use ubuntu:22.x for version). Since these rocks will fall out of support soon, should we merge this PR in red knowing the changes in this PR should not affect any of the rocks functionality?

**Review by misohu — APPROVED** [review]
Agreed lets merge it :)

**Pull request merged**